### PR TITLE
Update service worker release version

### PR DESCRIPTION
**Pull request by jlmonroy13**

### CODE REVIEW

**Review by dmanjunath — CHANGES_REQUESTED** [review]
unfortunately we also have to manually make changes to index.html and the css/js in the build/ directory. would you mind copying over those changes as well?

File: .github/workflows/generate-service-worker.yml [hunk 1]
```diff
@@ -18,13 +18,18 @@ jobs:
...
-          docker pull earthfast/service-worker:v0.10.0
+          docker pull earthfast/service-worker:v0.11.0
           mkdir -p build
-          docker create --name temp_container earthfast/service-worker:v0.10.0 /bin/true
+          docker create --name temp_container earthfast/service-worker:v0.11.0 /bin/true
+          # Extract main.js.tmpl and rename it to earthfast-sw.js
           docker cp temp_container:/earthfast-sw/dist/templates/main.js.tmpl build/earthfast-sw.js
+          # Extract the earthfast folder
+          docker cp temp_container:/earthfast-sw/dist/earthfast build/
+          # Extract index.html
```
Comment (dmanjunath) [inline .github/workflows/generate-service-worker.yml:31]: I think we also have to copy the css/js files

Comment (jlmonroy13) [reply .github/workflows/generate-service-worker.yml:31]: are they not inside earthfast folder?

File: .github/workflows/generate-service-worker.yml [hunk 2]
```diff
@@ -18,13 +18,18 @@ jobs:
         with:
           path: service-worker
 
-      - name: Extract main.js.tmpl from Docker image
+      - name: Extract files from Docker image
         working-directory: ${{ github.workspace }}/service-worker
         run: |
-          docker pull earthfast/service-worker:v0.10.0
+          docker pull earthfast/service-worker:v0.11.0
```
Comment (dmanjunath) [inline .github/workflows/generate-service-worker.yml]: can we change this to v0.12.0

File: .github/workflows/generate-service-worker.yml (hunk 1 above)
Comment (dmanjunath) [reply .github/workflows/generate-service-worker.yml:31]: They are but you only do docker cp for the index.html. You might be able to do /* for all files in that directory

Comment (dmanjunath) [reply .github/workflows/generate-service-worker.yml:31]: Oh wait I see what's happening. You extract the folder first and then you extract the index.html after that. That might work

**Review by dmanjunath — APPROVED** [review]

**Pull request merged**